plugins/sidebar/overlays/project.js

```diff
@@ -8,58 +9,20 @@ const TextField = require('react-material/components/TextField');
 const Project = require('../project');
 const ProjectList = require('../project-list');
 
+const ProjectActions = require('../../../src/actions/ProjectActions.js');
+const ProjectStore = require('../../../src/stores/ProjectStore.js');
```

Comment:
stores are instances of classes, convention says they should start with a lowercase letter